Solutions/IoTOTThreatMonitoringwithDefenderforIoT/Playbooks/MailBySensor/MailBySensor.json

```diff
@@ -238,6 +238,11 @@
                      "connectionName":"[variables('AzureSentinelConnectionName')]",
                      "id":"[concat('/subscriptions/', subscription().subscriptionId, '/providers/Microsoft.Web/locations/', resourceGroup().location, '/managedApis/azuresentinel')]"
                   },
+				  "azuremonitorlogs": {
```

Comment:
This is hardcoded. Can you please use the concat function as in line number 239.
Thanks

Reply:
resolved the comments. Thanks